### PR TITLE
Destroy deployment after PR is closed

### LINKED ISSUE #1912: [8pt] Push any branch to the test server on demand
It's quite challenging to check how a documentation version from a branch looks like.

Need to create github actions workflow which will upload documentation to some s3 bucket and trigger downloading it by the test server.

### PR DESCRIPTION
**Pull request by artembo**
#1912

### CODE REVIEW

**Comment by artembo** [comment]
Unfortunately, couldn't find a way to perform this workflow only for labeled as "deploy-branch" PRs. It will run on each pull request is closed.

File: .github/workflows/destroy-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+on:
+  pull_request:
+    types:
+      - closed
+jobs:
+  destroy-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
```
Comment (NickVolynkin) [inline .github/workflows/destroy-deployment.yml:12]: What would happen if one opens a link to the destroyed deployment?

**Comment by NickVolynkin** [comment]
> Unfortunately, couldn't find a way to perform this workflow only for labeled as "deploy-branch" PRs. It will run on each pull request is closed.

Is our backend okay with reqrests to delete a nonexisting version (deployment)?

File: .github/workflows/destroy-deployment.yml (hunk 1 above)
Comment (artembo) [reply .github/workflows/destroy-deployment.yml:12]: The page will be redirected to the last version

**Comment by artembo** [comment]
> > Unfortunately, couldn't find a way to perform this workflow only for labeled as "deploy-branch" PRs. It will run on each pull request is closed.
> 
> Is our backend okay with requests to delete a nonexisting version (deployment)?

Yep, it will just ignore creating the task end returns 404

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**